packages/hash/frontend/src/blocks/page/collab/EditorConnection.ts

```diff
@@ -145,12 +150,7 @@ export class EditorConnection {
       const requestDone = "requestDone" in action && action.requestDone;
 
       let sendable;
-      if (newEditState.doc.content.size > 40000) {
-        if (this.state.comm !== "detached") {
-          this.report.failure(new Error("Document too big. Detached."));
-        }
-        this.state = new State(newEditState, "detached", nextVersion);
-      } else if (
```

Comment:
Yup – removing one of the potential reasons collab might stop working, instead of adding support for showing the modal when this happens. This limitation comes from the original demo and I think is there just to help stop people from overloading their demo server.